languages/elixir/exercises/concept/numbers/test/assembly_line_test.exs

```diff
@@ -0,0 +1,53 @@
+defmodule AssemblyLineTest do
+  use ExUnit.Case
+
+  test "production rate per hour for speed 0" do
+    rate = AssemblyLine.production_rate_per_hour(0)
+    assert is_float(rate) and rate == 0.0
```

Comment:
It's worth noting that `is_float(rate) and rate == 0.0` could be expressed as simply `rate === 0.0`. Might be worth writing it this way once, noting in a comment that these two are equivalent, then moving to the more concise `===` assertion.

Reply:
Agreed, let's change one to `===` so that people can see that they are functionally the same, and can then search for the function to read about it's deatils?

Reply:
See below, on further thinking, `===` produces better test feedback in the failing case.

Reply:
These will be updated in next commit.